docs/known_bugs.rst

```diff
@@ -111,11 +111,6 @@ try a different OS/browser combo.
 Specviz
 -------
 
-Collapse Plugin spectral bounds do not match selected region
```

Comment:
This entry is misleading because Specviz does not load Collapse and there is already a similar entry above for Cubeviz.